scrunch/datasets.py

```diff
@@ -319,14 +420,30 @@ def __repr__(self):
     def __str__(self):
         return self.name
 
+    @property
+    def members(self):
+        return Members(self.resource)
+
+    def get_dataset(self, dataset):
+        try:
+            shoji_ds = self.resource.datasets.by('name')[dataset].entity
+        except KeyError:
+            try:
+                shoji_ds = self.resource.datasets.by('id')[dataset].entity
```

Comment:
Factor out `self.resource.datasets` here since you are doing that GET request two times.